### ISSUE TITLE
Promise returned in function argument where a void return was expected.

### ISSUE THREAD

**Issue by manusa** [manusa/electronim#345]
## Description

The following code has been marked as a [bug](https://sonarcloud.io/project/issues?open=AYh_YKv_W2wJeQETzpxd&id=manusa_electronim):

https://github.com/manusa/electronim/blob/f246881f34479a0f14780c292345cfd9c1b2f04e/src/main/quit.js#L31

It should be changed to:

```javascript
  [...persistentSessions, session.defaultSession].forEach(s => {
    // eslint-disable-next-line no-console
    clearCache(s).catch(console.error);
  });
```

**Comment by bharath090909** [comment]
@manusa can i take up on this issue ?

**Comment by manusa** [comment]
> @manusa can i take up on this issue ?

Sure!

**Comment by Abhishekgupta204** [comment]
Please assign this issue to me @manusa

**Comment by bharath090909** [comment]
@manusa Task completed and raised a PR please check.

**Comment by manusa** [comment]
> Please assign this issue to me @manusa

Hi @Abhishekgupta204


@bharath090909 is currently working on this, if they don't complete it I'll hand it over to you.

**Comment by bharath090909** [comment]
@manusa i have raised a new PR please check and let me know if any changes.

**Comment by manusa** [comment]
Fixed by #348
